WooCommerce/Classes/Authentication/Prologue/LoginOnboardingPageViewController.swift

```diff
@@ -0,0 +1,98 @@
+import UIKit
+
+/// Displays the Login Prologue carousel, populated with `LoginOnboardingPageTypeViewController` pages.
+///
+final class LoginOnboardingPageViewController: UIPageViewController {
```

Comment:
While Github marks this file as new file, it's not, it's just an issue with how Github renders the diff (an ongoing [issue](https://github.com/orgs/community/discussions/8573)), for an easier review, please exclude the initial commit (the rename commit).